features/foutafhandeling.feature

```diff
@@ -0,0 +1,135 @@
+        | name             | code   | reason                |
+        | <parameter>[min] | <code> | <reason omschrijving> |
+
+        Voorbeelden:
+        | path                   | query string                               | parameter   | code  | reason omschrijving              |
+        | /adresseerbareobjecten | ?oppervlakte[min]=200&oppervlakte[max]=100 | oppervlakte | range | min mag niet hoger zijn dan max. |
+        | /panden                | ?bouwjaar[min]=2000&boujaar[max]=1000      | bouwjaar    | range | min mag niet hoger zijn dan max. |
+
+Rule: waarde van parameter in request body is valide
+
+    Abstract Scenario: waarde van een parameter in de request body is niet correct
+        Als '<path><request body>' wordt aangeroepen
+        Dan bevat de response de volgende velden
+        | naam   | waarde                                        |
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | code   | paramsValidation                              |
+        | detail | <detail>                                      |
+        En bevat de response de volgende invalidParams
+        | name        | code   | reason                |
+        | <parameter> | <code> | <reason omschrijving> |
+
+        Voorbeelden:
+        | path      | request body                                                                                                                             | parameter            | code             | reason omschrijving                      |
+        | /adressen | AdresBody.geometrie.intersects=POLYGON ((a b,c d))                                                                                       | geometrie.intersects | geometryFormat   | Waarde is niet conform {format} formaat. |
+        | /adressen | AdresBody.geometrie.intersects=POLYGON ((134647.123))                                                                                    | geometrie.intersects | geometryFormat   | Waarde is niet conform {format} formaat. |
+        | /adressen | AdresBody.geometrie.intersects=POLYGON ((134647.123 457842.456, 134747.789 d))                                                           | geometrie.intersects | geometryFormat   | Waarde is niet conform {format} formaat. |
+        | /adressen | AdresBody.geometrie.intersects=POLYGON (([meer dan 350 coördinaten]))                                                                    | geometrie.intersects | maxItems         | Array bevat meer dan {maxItems} items.   |
+        | /adressen | AdresBody.geometrie.intersects=POLYGON ((5.96072575 52.18450437,5.95270058 52.17811002, 5.96793552 52.17816266, 5.96072575 52.18450437)) | geometrie.intersects | geometryMismatch | Waarde is niet conform opgegeven CRS.    |
+
+    Abstract Scenario: waarde van AdresBody.geometrie.intersects parameter overschrijdt maximale toegestane oppervlakte 
```

Comment:
kan je dit niet functioneler opschrijven? Bijvoorbeeld "waarde van opgegeven contour overschrijdt maximaal toegestane oppervlakte"

Reply:
is dit geen aparte rule? valt nu onder "Rule: waarde van parameter in request body is valide", maar in de specificaties staat niet dat er een maximaal oppervlakte is. 

Dus ik denk dat óf aan de specificaties (GeometrieWKTFilter description) moet worden toegevoegd dat er een maximaal oppervlakte is, óf dit onder een aparte rule (bij zoeken met contour mag het oppervlakte van de contour niet groter dan ..km2 zijn) moet worden beschreven.

Reply:
dit is geen abstract scenario

Reply:
Aangepast. 
Vraag over maximale oppervlakte staat nog uit.
Vraag over maximum aantal coördinaten staat nog open.